remoting/rico-remoting-server-spring-test/src/main/java/module-info.java

```diff
@@ -14,4 +14,9 @@
     requires static java.servlet;
     requires testng;
```

Comment:
if testng is no longer used, then please remove it

Reply:
Would like me to delete everything relating to testng? Such as the SpringTestNGControllerTest?